addons/validate_mission/fn_run.sqf

```diff
@@ -156,6 +156,72 @@ private _check_cba_settings = {
     ["Changed CBA settings", true, [], _strings];
 };
 
+private _check_playable_units = {
+    private _msg = [];
+    private _pass = true;
+    if (count playableUnits < 35) then {
+        (_msg pushBack "Not enough playable units: 35 required")};
+        _pass = false;
+    if ((group player get3DENAttribute "a3aa_ee_persistent_callsign" select 0) isEqualTo "") then {
+        _msg pushback "Possible AI unit marked as player.";
+        _msg pushback "Did you place some AI before any playable units?";
+        _pass = false;
+    };
+
+    ["Playable unit checks", _pass, [], _msg]
+};
+
+private _check_default_modules = { 
+    private _msg = []; 
+    private _pass = true; 
+    if ((all3DENEntities#7) apply {(_x get3DENAttribute "name")#0} find "CNTO" isEqualTo -1) then { 
+        _msg pushBack "Default CNTO modules not detected.";
+        _msg pushBack "Place down the default CNTO Co-op or PvP modules.";
+        _pass = false; 
+
+    } else { 
+
+        private _briefingModules = all3DENEntities#3 select {typeOf _x isEqualTo "a3aa_ee_briefing"}; 
+        if (count _briefingModules > 0) then { 
+
+            { 
+                private _module = _x;
+                private _moduleIndex = _forEachIndex + 1;
+                { 
+                    private _fieldText = (_module get3DENAttribute _x)#0; 
+                    if (count _fieldText < 5) then { 
+                        _msg pushBack format ["Field '%1' of the OPORD briefing module %2 is incomplete", _x select [17], _moduleIndex]; 
+                        _pass = false; 
+                    } 
+                } forEach ["a3aa_ee_briefing_situation", "a3aa_ee_briefing_mission", "a3aa_ee_briefing_execution", "a3aa_ee_briefing_admin_logistics"]; 
+            } forEach _briefingModules; 
+
+        } else { 
+            _msg pushBack "No OPORD briefing modules found";
+            _msg pushBack "The default CNTO modules seem to have been placed, yet no briefing module has been found.";
+            _msg pushBack "Delete the whole CNTO Module set and re-place it.";
+            _pass = false; 
+        } 
+
+    }; 
+
+    ["CNTO module check", _pass, [], _msg]; 
```

Comment:
Might want to actually return the briefing modules with empty fields here, instead of `[]`. The validate mission framework will then display them in a list, with positions, etc., so you don't need to list them one-by-one in the message.